src/routes/lobby/components/LobbyPlayerIndicator.vue

```diff
@@ -126,6 +129,34 @@ export default {
   z-index: 1;
 }
 
+.water-container {
+  top: 0;
+  max-height: 33vh;
+  max-width: calc(35vh / 1.45);
+  overflow: hidden;
+  background-color: transparent;
+  border-radius: 10px;
+  position: relative;
+  -webkit-clip-path: polygon(0 0, 100%, 0, 90% 100%, 10% 100%);
+  clip-path: polygon(0 0, 100%, 0, 90% 100%, 10% 100%);
+}
+.water {
+  width: 3600px;
+  height: 800px;
+  background-image: url('../../../../public/img/waves.svg');
```

Comment:
Images should just use the `/img` folder directly, which should work for local development via vite and for production via sails
```suggestion
  background-image: url('/img/waves.svg');
```

Reply:
You can see the examples of background-image css in `GameView` for the game board image:
```
#game-view-wrapper {
  color: #fff;
  width: 100vw;
  height: 100%;
  background-image: url('/img/game/board-background.svg');
```

Reply:
Will get fixed in the next commit!